.github/workflows/cypress.yml

```
@@ -20,7 +20,11 @@ jobs:
       - name: Cypress run
         uses: cypress-io/github-action@v1
         with:
+          record: true
           build: npm run build
+        env:
+          CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

```suggestion
        env:
          CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
        with:
          record: true
          build: npm run build
```
The original order has various 'with' properties being interpreted as being part of  'env'